tests/training/metrics/test_fbeta_multi_label_measure.py

```diff
@@ -0,0 +1,291 @@
+from typing import List
+
+import pytest
+import torch
+from sklearn.metrics import precision_recall_fscore_support
+from torch.testing import assert_allclose
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.common.testing import AllenNlpTestCase, multi_device
+from allennlp_multi_label.training.metrics import FBetaMeasureMultiLabel
+
+
+class TestFBetaMeasureMultiLabel(AllenNlpTestCase):
+    def setup_method(self):
+        super().setup_method()
+        self.predictions = torch.tensor(
+            [
+                [0.55, 0.25, 0.10, 0.10, 0.20],
+                [0.10, 0.60, 0.10, 0.95, 0.00],
+                [0.90, 0.80, 0.75, 0.80, 0.00],
+                [0.49, 0.50, 0.95, 0.55, 0.00],
+                [0.60, 0.49, 0.60, 0.65, 0.85],
+                [0.85, 0.40, 0.10, 0.20, 0.00],
+            ]
+        )
+        self.targets = torch.tensor(
+            [
+                [1, 1, 0, 0, 0],
+                [0, 1, 0, 1, 0],
+                [1, 1, 0, 1, 0],
+                [1, 1, 1, 1, 0],
+                [1, 1, 1, 1, 0],
+                [0, 0, 0, 0, 0],
+            ]
+        )
+
+        # detailed target state
+        self.pred_sum = [4, 3, 3, 4, 1]
+        self.true_sum = [4, 5, 2, 4, 0]
+        self.true_positive_sum = [3, 3, 2, 4, 0]
+        # I don't think this is correct for multi-label
+        self.true_negative_sum = [10.0, 10.0, 12.0, 11.0, 14.0]
```

Comment:
Why are these floats ?

Reply:
I don’t remember but in any case they are wrong. I think to code that computes this comes from the parent class and works for the multi-class (not multi-label) setting.

Reply:
yup, I think we never changed the true_negative_sum in our code.

Reply:
Gotcha. I think for completeness sake we should update the method that computes `true_negative_sum` for the multi-label case.

Reply:
yeah for sure, I don't think it should be that difficult to do, and It's definitely a prereq if we want to merge this to Allen. We should still rename it to F1MeasureMultilabel or add the beta param option.

Reply:
The beta param option does exist in our code, is that what you mean?

https://github.com/semantic-health/allennlp-multi-label/blob/f37fbc369cb1a30c3c62981844fab0acc877b433/allennlp_multi_label/training/metrics/fbeta_multi_class_measure.py#L64-L70

Reply:
Oh yeah I just checked it out, it should totally work without issue. I completely forgot I did that.